examples/jsm/loaders/LDrawLoader.js

```diff
@@ -2351,45 +2354,44 @@ class LDrawLoader extends Loader {
 
 		}
 
+		material.color.setStyle( color, COLOR_SPACE_LDRAW );
 		material.transparent = isTransparent;
 		material.premultipliedAlpha = true;
 		material.opacity = alpha;
 		material.depthWrite = ! isTransparent;
-		material.color.convertSRGBToLinear();
 
 		material.polygonOffset = true;
 		material.polygonOffsetFactor = 1;
 
 		if ( luminance !== 0 ) {
 
-			material.emissive.set( material.color ).multiplyScalar( luminance );
+			material.emissive.setStyle( material.color, COLOR_SPACE_LDRAW ).multiplyScalar( luminance );
```

Comment:
Here, `material.color` is a `THREE.Color`. Is this correct?

Reply:
No that's a mistake, thanks!